src/cli/cli.cc

```diff
@@ -45,7 +45,7 @@ static const std::unordered_map<InstallResult::Status, StatusCode> i2s = {
     {InstallResult::Status::NeedsCompletion, StatusCode::InstallNeedsReboot},
     {InstallResult::Status::AppsNeedCompletion, StatusCode::InstallAppsNeedFinalization},
     {InstallResult::Status::BootFwNeedsCompletion, StatusCode::InstallNeedsRebootForBootFw},
-    {InstallResult::Status::DownloadFailed, StatusCode::InstallAppPullFailure},
+    {InstallResult::Status::DownloadFailed, StatusCode::DownloadFailure},
```

Comment:
It doesn't indicate whether its ostree or app pull failure.

Reply:
I've added a new Status for the ostree download failure, reverting the change on `{InstallResult::Status::DownloadFailed, StatusCode::InstallAppPullFailure}` mapping.